syntax = "proto3";

package kmsp11;

message TestMessage {
  string string_value = 1;
  int32 int32_value = 2;
}
